lib/ZnapZend/ZFS.pm

```diff
@@ -1321,13 +1321,13 @@ sub getSnapshotProperties {
     }
 
     if (!defined($inherit)) {
-        $inherit = new inheritLevels;
+        $inherit = ZnapZend::InheritLevels->new;
         $inherit->zfs_local(1);
     } else {
         # Data type check
-        if ( ! $inherit->isa('inheritLevels') ) {
-            $self->zLog->warn("getSnapshotProperties(): inherit argument is not an instance of struct inheritLevels");
-            my $newInherit = new inheritLevels;
+        if ( ! ($inherit->isa('ZnapZend::InheritLevels') or $inherit->isa('InheritLevels')) ) {
```

Comment:
why do you allow both ? can there still be InheritLevels

Reply:
Good catch. Was not sure which sort of name it reacts to, so while developing - kept both to make sure the change is viable at all :\